docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
     image: tootsuite/mastodon
     restart: always
     env_file: .env.production
-    command: bundle exec rails s -p 3000 -b '0.0.0.0'
+    command: /mastodon/docker_entrypoint.sh bundle exec rails s -p 3000 -b '0.0.0.0'
```

Comment:
See what was undone in be9bab171dc2b1fe43bc742decb71f64541ca347: the entrypoint does not need to be hardcoded in docker-compose.yml

Reply:
It does because we do not want to use the entrypoint for `docker-compose run`. It should only be used when running a server process.

Reply:
People who don't use docker-compose.override.yml method will get merge conflicts from this change

Also, strictly speaking the streaming API does not need any directory permissions, I think.

What if someone runs using Docker but not docker-compose? People only need to chown once per installation lifetime, so I think that's a simpler solution than re-adding an entrypoint in this way...

Reply:
> People who don't use docker-compose.override.yml method will get merge conflicts from this change

The documentation suggests only to modify the file for volumes, and such changes will not conflict with this change.

> Also, strictly speaking the streaming API does not need any directory permissions, I think.

The script is used to change the permission from `root` to `mastodon`.

> What if someone runs using Docker but not docker-compose? People only need to chown once per installation lifetime, so I think that's a simpler solution than re-adding an entrypoint in this way...

If there is any people who runs without docker-compose, they should be able to solve their own problems by themselves. They are not following the documentation anyway.

> People only need to chown once per installation lifetime, so I think that's a simpler solution than re-adding an entrypoint in this way...

Why not make it simple for people trying out new installation rather than for people editing much of `docker-compose.yml` or managing containers without docker-compose?